build/Makefile

```diff
@@ -13,12 +13,12 @@
 
 # Define core directory below which everything resides. This is the
 # parent directory of the 'build' directory
-F_MASTER =
+F_MASTER = /home/wayandn/summa
```

Comment:
Fixed. Sorry, I had assumed after I issued a PR it would somehow freeze at that commit, but I understand now that it always points to this branch.